bin/gitlab-backup

```diff
@@ -339,7 +343,7 @@ def main():
         log_info('Create output directory {0}'.format(output_directory))
         mkdir_p(output_directory)
 
-    items = client.projects.list(as_list=False)
+    items = client.projects.list(as_list=False,owned=args.owned_only)
```

Comment:
Can you add a whitespace after the comma?

Reply:
Done.